tests/e2e/vsphere.go

```diff
@@ -1069,3 +1069,15 @@ func (vs *vSphere) verifyLabelsAreUpdated(volumeID string, matchLabels map[strin
 	}
 	return nil
 }
+
+//verifyDatastoreMatch verify is dsurl matches with given one for the volumeid
+func (vs *vSphere) verifyDatastoreMatch(volumeID string, dsUrl string) bool {
```

Comment:
Please take an array for datastoreUrl, if one or more datastores are tagged/available to consume, volume can go to any of the tagged datastores.

Reply:
I think we should write wrapper over this method for your usecase, instead of modifying this one.